tests/fuzzing/llvm_fork_env_fuzz_test.py

```diff
@@ -33,26 +34,29 @@ def test_fuzz(env: LlvmEnv):
             break
     else:
         # Fork the environment and check that the states are equivalent.
-        new_env = env.fork()
+        fkd = env.fork()
         try:
             print(env.state)  # For debugging in case of error.
-            assert env.state == new_env.state
+            assert env.state == fkd.state
             # Check that environment states remain equal if identical
             # subsequent steps are taken.
             for _ in range(POST_FORK_ACTIONS):
                 action = env.action_space.sample()
                 observation_a, reward_a, done_a, _ = env.step(action)
-                observation_b, reward_b, done_b, _ = new_env.step(action)
+                observation_b, reward_b, done_b, _ = fkd.step(action)
 
                 print(env.state)  # For debugging in case of error.
-                np.testing.assert_array_almost_equal(observation_a, observation_b)
-                np.testing.assert_almost_equal(reward_a, reward_b)
                 assert done_a == done_b
+                np.testing.assert_array_almost_equal(observation_a, observation_b)
```

Comment:
Why `almost_equal`? None of the observations are noisy, yet, right?

Reply:
Ah, good catch, force of habit :)